assignment-0/main.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+func main() {
+	var (
+		config *rest.Config
+		err    error
+	)
+	kubeconfig := flag.String("kubeconfig", "/home/pratik/.kube/config",
+		"location for kubeconfig file")
```

Comment:
`/home/pratik` would not be there on my machine.

Reply:
fixed